core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -99,6 +99,7 @@ export class TranslationModalComponent {
   hadCopyParagraphError = false;
   hasImgTextError = false;
   hasIncompleteTranslationError = false;
+  elementViolation: string;
```

Comment:
I think it would be a slightly better user experience if this was a list so that we could show all the element violations at once?

Reply:
Done